packages/stream/demo/index.tsx

```diff
@@ -0,0 +1,114 @@
+import { render } from "solid-js/web";
+import { Component, createEffect, createSignal } from 'solid-js';
+import { createStream } from "../src";
+
+declare module "solid-js" {
+  namespace JSX {
+    interface ExplicitProperties {
+      srcObject?: MediaStream
+    }
+  }
+}
+
+const iceservers = {
+  iceServers: [
+    {
+      urls: ['stun:stun1.l.google.com:19302', 'stun:stun2.l.google.com:19302'],
+    },
+  ],
+  iceCandidatePoolSize: 10,
+}
+
+const App: Component = () => {
+  const [stream, { mutate, stop }] = createStream({ video: true, audio: true })
+  const [remoteStream, setRemoteStream] = createSignal<MediaStream>()
+
+  const [ICE, setICE] = createSignal("")
+  const [input, setInput] = createSignal("")
+  
+  let PeerConnection = new RTCPeerConnection(iceservers)
+  
+  PeerConnection.ontrack = (event) => {
+    setRemoteStream(event.streams[0])
+  }
+
+  PeerConnection.onicecandidate = () => {
+    setICE(JSON.stringify(PeerConnection.localDescription))
+  }
+  
+  createEffect(() => {
+    const RawStream = stream()
+    if (RawStream === undefined) return
+    RawStream.getTracks().forEach(track => PeerConnection.addTrack(track, RawStream))
+  })
+  
+  async function StartCall() {
+    if (stream() === undefined) return alert("local Stream not ready yet")
+
+    if(PeerConnection.localDescription !== null) return
+    const offer = await PeerConnection.createOffer()
+    await PeerConnection.setLocalDescription(offer)
+  }
+
+  async function AnswerCall() {
+    if (stream() === undefined) return alert("local stream not ready yet")
```

Comment:
Wouldn't it be better to disable the button if the stream is not available instead of blocking the UI with an alert?